core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3005,6 +3033,7 @@ private void journalSetAttribute(LockedInodePath inodePath, long opTimeMs,
 
   @Override
   public void scheduleAsyncPersistence(AlluxioURI path) throws AlluxioException {
+    checkUfsMode(path, OperationType.WRITE);
```

Comment:
Is this the latest we can do this, or can we wait until the file is about to be persisted?